tests/helpers.py

```diff
@@ -1,5 +1,13 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField
+
+
 def login(user, testapp):
     res = testapp.post(
         "/", params={"email": user.email, "password": user.test_password}
     ).follow()
     assert "Přihlášení proběhlo úspěšně" in res
+
+
+class TestingForm(FlaskForm):
```

Comment:
Tady je potřeba změnit jméno třídy na něco, co nezačíná na Test, protože jinak to pytest považuje za testovací třídu a vypíše následující:
```
tests/helpers.py:12
  /github/workspace/tests/helpers.py:12: PytestCollectionWarning: cannot collect test class 'TestingForm' because it has a __init__ constructor (from: tests/test_functional.py)
    class TestingForm(FlaskForm):
```
Tedy je to považováno za třídu s testy, ale protože má zděděný `__init__`, tak se přeskočí a vygeneruje warning. Stačí jiné jméno a bude to v pohodě.

Reply:
Přejmenováno.